src/main/java/step1/Calculator.java

```diff
@@ -0,0 +1,83 @@
+package step1;
+
+public class Calculator {
+
+    public Calculator(){
+
+    }
+    //더하기
+    private int _sum(int a, int b) {
+        return a + b;
+    }
+
+    //뺄셈
+    private int _minus(int a, int b) {
+        return a - b;
+    }
+
+    //곱셈
+    private int _multiply(int a, int b) {
+        return a * b;
+    }
+
+    //나눗셈
+    private int _divide(int a, int b) {
+        return (int) (a / b);
+    }
+
+
+    private int _changeInt(String numStr){
+        return Integer.parseInt(numStr);
+    }
+
+
+    private int _executeExp(int num,int resultNum,String sign){
+
+        if("+".equals(sign)){
+            return _sum(resultNum,num);
+        }
+
+        if("-".equals(sign)){
+            return _minus(resultNum,num);
+        }
+
+        if("*".equals(sign)){
+            return _multiply(resultNum,num);
+        }
+
+        if("/".equals(sign)){
+            return _divide(resultNum,num);
+        }
+
+        return -1;
+    }
+
+
+    private String[] _divideBlank(String expression){
+        return expression.split(" ");
+    }
+
+    public int execute(String expression) {
+
+        //빈공백을 중심으로 수식 쪼개기
+        String [] dividedExpArr = _divideBlank(expression);
+        int dividedExpArrSize = dividedExpArr.length;
+        int resultNum = Integer.parseInt(dividedExpArr[0]);
+        for(int i=1; i<dividedExpArrSize; i+=2){
+
+            int num = Integer.parseInt(dividedExpArr[i+1]);
```

Comment:
단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리되어져야 합니다 :)
```suggestion
        for (int i = 1; i < dividedExpArrSize; i += 2) {
            int num = Integer.parseInt(dividedExpArr[i + 1]);
```